### PR TITLE
Translation of Code of Conduct to the Czech language

### PR DESCRIPTION
**Pull request by lucieva**
Hi all, I translated the Code of Conduct to Czech language.

### CODE REVIEW

**Comment by hagenburger** [comment]
@lucieva awesome work! Thanks a lot for your efforts! 

I’d love to get some reviews of Czech native speakers and then go for it.

**Comment by skade** [comment]
Tweeted a request for review help: https://twitter.com/Argorak/status/1346448236911984642

**Comment by petrsvihlik** [comment]
I'll take a look. You can assign me as a reviewer.

**Comment by skade** [comment]
@petrsvihlik sadly, reviewers cannot be external. We'll keep that as a note :)

File: source/localizable/_code-of-conduct.cz.md
Comment (petrsvihlik) [reply source/localizable/_code-of-conduct.cz.md:36]: This one is very contextual. I think "sledování" captures the meaning slightly better, however, it may be best to leave it out completely and use just "pronásledování".

**Review by petrsvihlik — CHANGES_REQUESTED** [review]
The translation overall is very good and reflects the linguistic nuances of the Czech language which makes the text sound natural and increases its readability.

**Comment by skade** [comment]
Ping @lucieva. I'm standing ready to merge once the changes are discussed/merged. :)

**Comment by lucieva** [comment]
Thanks for the review @petrsvihlik. You can merge the changes @skade

**Comment by chastell** [comment]
Thanks so much for your work, @lucieva and @petrsvihlik! Merged and deployed. ✅

**Pull request merged**
